ruby-challenges.rb

```diff
@@ -33,3 +65,15 @@
 # Expected output example: my_bike.pedal_faster(18) => 28
 # Expected output example: my_bike.brake(5) => 23
 # Expected output example: my_bike.brake(25) => 0
+
+
+new_bike = Bike.new 'Trek', 2, 0
+p new_bike
+new_bike.my_bike 'Trek', 2, 0
+
+p new_bike.pedal_faster 10
+p new_bike.pedal_faster 18
+
+p new_bike.stop 5
+
+# I spent more than 2 hours in this and I could not get it to work, I get nil for brake. I tried to change the syntax of += and add and else statment equal to zero but I just get zero. I followed the exampled from the syllabus, challenges and the notes. 
```

Comment:
Good work! There is a line of code which is a little math that we would want to bring in. Maybe we can address it durning examples or office hours!